### DISCUSSION TITLE
Custom connectors

### DISCUSSION THREAD

**Discussion by kotyara85** [dexidp/dex#2455] [Q&A] [answered]
Does dex support custom connectors?
We want to use dex to sign tokens.
Our wofkflow:
our app -> dex -> idp -> (dex token) -> our system which will verify dex token in dex server -> return claims which dex will use to sign a secondary token
That sounds complicated, but we need this to issues service account token which come along with actual user tokens
Thanks

**Comment by nabokihms** [comment, marked as answer]
@kotyara85 , thank you for the question. At the current state, Dex does not support custom connectors.

We have an issue about it here. https://github.com/dexidp/dex/issues/1907
